### PR TITLE
Update cloud support for phone number assignment cmdlets

### PR DESCRIPTION
**Pull request by JensTrier**
Noting that cmdlets are only supported in commercial and GCC cloud instances.

Removing preview note from Get cmdlet.

Adding a new example to Get cmdlet.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md [hunk 1]
```diff
@@ -267,7 +271,8 @@ Accept wildcard characters: False
 ```
 
 ### -TelephoneNumberContains
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-CsPhoneNumberAssignment.md:273]:
```suggestion
### -TelephoneNumberContain

```

**Review by get-itips — APPROVED** [review]
I am ok after the suggestion

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**